charts/seed-controlplane/charts/kube-apiserver/templates/network-policy.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: kube-apiserver-deny-blacklist
+  namespace:  {{ .Release.Namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      app: kubernetes
+      role: apiserver
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - ipBlock:
+        # Allow all except seed networks
+        cidr: 0.0.0.0/0
+        except:
+        - {{ .Values.seedNetworks.pod }}
+        - {{ .Values.seedNetworks.node }}
+        - {{ .Values.seedNetworks.service }}
+        - {{ .Values.cloudProviderMetadataServiceIP }}/24
```

Comment:
1. It should be `/32` for single IP
1. Put the range in `cloudProviderMetadataServiceIP` so we don't hardcode `/24`
1. Change the name to `cloudProviderMetadataServiceCIDR`